src/Radio/RadioGroup.js

```diff
@@ -15,131 +15,83 @@
  */
 
 /* @flow */
-import React, { Children, cloneElement, createElement } from 'react';
-import { createStyledComponent } from '../styles';
+import React from 'react';
+import { createThemedComponent, mapComponentThemes } from '../themes';
+import ChoiceGroup, {
+  componentTheme as choiceGroupComponentTheme
+} from '../Choice/ChoiceGroup';
 import Radio from './Radio';
 
 type Props = {
   /**
-   * Checked state of the radio button. Primarily for use with controlled
-   * components. If this prop is specified, an `onChange` handler must also be
-   * specified. See also: `defaultChecked`.
+   * Value of the selected Radio; primarily for use with controlled components.
+   * If this prop is specified, an `onChange` handler must also be specified.
+   * See also: `defaultChecked`.
    */
   checked?: string,
   /** Mineral Radio components */
```

Comment:
We typically link component names when used in this manner.

Reply:
Updated.